server/etc/e-smith/templates/var/lib/nethserver/sieve-scripts/before.sieve/10junkmail

```diff
@@ -16,13 +16,14 @@ if header :contains "X-Spam-Flag" "YES" {
 
 # Rspamd is probably not installed, the subject sieve
 # script is activated if SpamSubjectPrefixString exists
+# and SpamSubjectPrefixStatus is enabled
```

Comment:
I think we can remove the comment block: it does not add more information. If not, reword it like "When Rspamd is installed..."